packages/valory/services/trader/service.yaml

```diff
@@ -79,7 +79,7 @@ type: skill
         creator_per_subgraph: &id003
           omen_subgraph: ${OMEN_CREATORS:list:["0x89c5cc945dd550BcFfb72Fe42BfF002429F46Fec"]}
         slot_count: ${SLOT_COUNT:int:2}
-        opening_margin: ${OPENING_MARGIN:int:300}
+        opening_margin: ${OPENING_MARGIN:int:86400}
```

Comment:
Please add this to the rest of the agents indexes in this configuration as well.

Reply:
fixed